src/test/java/com/neo4j/docker/TestConfSettings.java

```diff
@@ -199,12 +198,57 @@ void testEnvVarsOverride() throws Exception {
         //Start the container
         container.setWaitStrategy(Wait.forHttp("/").forPort(7474).forStatusCode(200));
         container.start();
-        //Read the config file to check if the config is not overriden
+        //Read the debug.log to check that dbms.memory.pagecache.size was set correctly
         Stream<String> lines = Files.lines(logMount.resolve("debug.log"));
         Optional<String> heapSizeMatch = lines.filter(s -> s.contains("dbms.memory.pagecache.size=512m")).findFirst();
         lines.close();
         Assertions.assertTrue(heapSizeMatch.isPresent(), "dbms.memory.pagecache.size was not set correctly");
         //Kill the container
         container.stop();
     }
+
+    @Test
+    void testEnterpriseConfigsAreSetCorrectly() throws Exception {
```

Comment:
The test name doesn't really describe what the test is checking. When I/you look back on this in 2 or 3 months time we'd have to read all the code to know what is actually being tested. Can you name it something like testEnterpriseOnlyDefaultsConfigsAreSet or something similar?